### PR TITLE
🚀 v5.2.1

### PR DESCRIPTION
**Pull request by julien-deramond**
## Specific to this version

- [x] The content of this v5.2.1 is defined and definitive
- [x] Merge the remaining PRs
  - [x] #1579 
  - [x] #1088 
  - [x] #1522
- [ ] Following `gh-pages` modifications have been done before merging this PR:
  - [ ] https://github.com/twbs/bootstrap/commit/5cff761c91cea600f410d88a3eff32241d2d3bcd
  - [ ] https://github.com/twbs/bootstrap/commit/c2b4f59e0c0e6de08510d51ef37d04cca41c22c6
- [ ] When merging to `gh-pages` check the diff with https://github.com/twbs/bootstrap/commit/1510239c8f6406ce07f50f5fbc1242767c07d397 

## Generic

- [x] Run linters;
- [x] Run compilers;
- [x] Run tests;
- [x] Check documentation site: examples and contents;
- [x] Test cross-browser compatibility locally and with [BrowserStack](https://www.browserstack.com/):
  - Firefox ESR
  - IE11 (v4 only)
  - Latest Edge, Chrome, Firefox, Safari
  - iOS Safari
  - Chrome & Firefox on Android
- [x] Including RTL mode;
- [x] Ask for reviews and accessibility testing;
- [x] [sync with Bootstrap](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/wiki/Syncing-with-Bootstrap)'s release and probably wait for it;
- [x] `npm run release-version $current_version $next_version` to bump version number
  - then, if bumping a minor or major version:
    - ~~Manually change `version_short` in `package.json`~~
    - ~~Add docs version to `site/data/docs-versions.yml`~~
    - ~~Manually change `docs_version` in `config.yml` and other references to the previous version~~
    - ~~Update redirects in docs frontmatter (`site/content/docs/_index.html`?)~~
    - ~~Move `site/content/docs/5.x` to `site/content/docs/5.x+1`~~
    - ~~Increment `site/static/docs/{version}` version~~
    - ~~Increment version in `nuget/boosted.nuspec`~~
    - ~~Ensure Algolia indexes new release content ([probably requires a PR](https://github.com/algolia/docsearch-configs/pull/1954))~~
    - ~~(Major version) Manually update the version in `nuget/boosted.nuspec` and `nuget/boosted.sass.nuspec`~~
  - check wrong matches in `CHANGELOG.md`, and maybe `site/content/docs/<version>/migration.md`
  - :warning: check the `package-lock.json` and `package.json` content, only "boosted" should have its version changed!
  - :warning: `site/content/docs/5.1/**/*.md` should not always be modified
- [x] if year changed recently, happy new year :tada: but please change © year in `.scss` main files (reboot, grid, utilities and main file) as well as in `NOTICE.txt`.
- [x] `npm run release` to compile dist, update SRI hashes in doc and package the release
- [x] Prepare changelog:
  - install [Conventionnal Changelog](https://github.com/conventional-changelog/conventional-changelog) and `conventional-changelog-cli` globally
  - run `conventional-changelog -p angular -i CHANGELOG.md -s`
  - and probably maintain [a ship list (eg for v4.4.0)](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/226)
- [x] commit and push `dist` with a `chore(release)` commit message
- [x] Manually run BrowserStack test
- [x] Manually run Percy test
- [x] merge (on `v4-dev` or `main`)
- [ ] tag your version, and push your tag
- [ ] [create a GitHub release](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/releases/new):
  - attach zip file
  - paste CHANGELOG / Ship list in the release's description
- [ ] Pack and publish
  - `npm pack`
  - if you are already logged in NPM (with a personnal account, for example), [you'd better use a repository scoped `.npmrc` file](https://stackoverflow.com/questions/30114166/how-to-have-multiple-npm-users-set-up-locally)
  - Publish:
    - if you're releasing a pre-release, use `--tag`, eg for v5-alpha1 `npm publish boosted-5.0.0-alpha1.tgz --tag next`
    - (v4 only) `npm publish --tag v4.x.y` (if you forgot and v4 becomes the latest version on NPM, you can run `npm dist-tag add boosted@5.x.y latest to fix it)
    - (v5 only) `npm publish`
- [ ] [publish on Nuget](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/wiki/Generate-NuGet-packages)
- [ ] check release on [NPM](https://www.npmjs.com/package/boosted), [Nuget](https://www.nuget.org/packages/boosted/), [Packagist](https://packagist.org/packages/orange-opensource/orange-boosted-bootstrap)…
- [ ] publish documentation on `gh-pages`:
  - [ ] copy `../_site` to the `gh-pages` branch
  - [ ] check every `index.html` used as redirections to be redirecting to the new release
  - [ ] when bumping minor version: ensure `dist` URLs in examples' HTML has changed
  - [ ] double-check everything before pushing, starting by searching for forgotten old version number occurences
- [ ] make an announcement in Plazza :tada:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,61 @@
...
+* **pagination:** remove CSS variables ([#1457](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1457)) ([92c1052](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/92c10527b036921fa3f27021f84e1dd986916d8a))
+* **paginations:** add missing breakpoints after lg ([#1579](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1579)) ([6255243](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/625524333d29c25fcb2fa4020b4e2c5c5c839fb7))
+* **stepped process:** improve rendering of arrow shapes ([#1485](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1485)) ([308a9f1](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/308a9f1d03e213b6ae2e3673c89d49c7b6c11231))
+* **tags:** handle close button rendering issues ([#1574](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1574)) ([7b3c494](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/7b3c49493b5d71b0e7355c2214c0d767a034b3ed))
```
Comment (louismaximepiton) [inline CHANGELOG.md:36]: Do we need this kind of commit ?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,61 @@
...
+* **pagination:** new `--#{$prefix}pagination-padding-end` CSS var ([#1416](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1416)) ([5a4f368](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/5a4f3688a33a9da3efc498c96bf5c1806df039d8))
+* **stepped process:** add dark variant ([#1434](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1434)) ([57633bb](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/57633bbcf90e6fe17e7b6503f67221eab4f4421e))
+* **utilities:** revamp `.img-thumbnail` and add desc in docs ([#1441](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1441)) ([344ab7e](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/344ab7ed5b8e31700a8eab3958ddfb069fbbb858))
+* **workflow:** temporarily deactivate BrowserStack ([#1547](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1547)) ([9dff4a1](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/commit/9dff4a1a6836ab79aca7dc48944b14767039974f))
+
+
+
 # [5.2.0](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/compare/v5.1.3...v5.2.0) (2022-07-27)
```
Comment (louismaximepiton) [inline CHANGELOG.md:61]: Weird display below, need to tackle anything in here ? (out of PR)

**Review by louismaximepiton — APPROVED** [review]
Few comments but fine for me

File: CHANGELOG.md (hunk 1 above)
Comment (julien-deramond) [reply CHANGELOG.md:36]: This file is generated automatically and I don't touch it. In the future, I'd prefer to use directly the content we create in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1429 or just have a list of the versions and the links to the releases that contain the precise release notes.

File: CHANGELOG.md (hunk 2 above)
Comment (julien-deramond) [reply CHANGELOG.md:61]: Nice catch. It is because of the `<select>` in the message that is interpretated! I'm fixing it.

**Pull request merged**